framework/celery_tasks/routers.py

```diff
@@ -14,6 +14,8 @@ def match_by_module(task_path):
             return CeleryConfig.task_high_queue
         if task_subpath in CeleryConfig.remote_computing_modules:
             return CeleryConfig.task_remote_computing_queue
+        if task_subpath in CeleryConfig.account_status_changes:
```

Comment:
this is either wrong or confusingly done -- `CeleryConfig.account_status_changes` is the string `"account_status_changes"`, and if that contains any `task_subpath` it seems coincidental

if you're doing this by updating `match_by_module` (tho there are other ways), pls do it consistently with the others -- add a list of module paths